### ISSUE TITLE
postgresql array Upsert Sql not support. 

### ISSUE THREAD

**Issue by siaron** [JSQLParser/JSqlParser#1110]
## SQL Paser
```
String UpsertSql = "UPSERT INTO SENSOR.RUN (l1, l2, l3) VALUES (ARRAY ['1','2'],'s2',ARRAY [3,4])";
Upsert parse = (Upsert) CCJSqlParserUtil.parse(UpsertSql);
```
## Exception
```
net.sf.jsqlparser.JSQLParserException: Encountered unexpected token: "," ","
    at line 1, column 64.

Was expecting one of:

    "&"
    "::"
    "<<"
    ">>"
    "COLLATE"
    "["
    "]"
    "^"
    "|"

	at net.sf.jsqlparser.parser.CCJSqlParserUtil.parseStatement(CCJSqlParserUtil.java:165)
	at net.sf.jsqlparser.parser.CCJSqlParserUtil.parse(CCJSqlParserUtil.java:59)
	at net.sf.jsqlparser.parser.CCJSqlParserUtil.parse(CCJSqlParserUtil.java:37)
	at com.nvr.hbase.PhoenixJDBCTest.test2(PhoenixJDBCTest.java:141)
	at com.nvr.hbase.PhoenixJDBCTest.main(PhoenixJDBCTest.java:55)
Caused by: net.sf.jsqlparser.parser.ParseException: Encountered unexpected token: "," ","
Caused by: net.sf.jsqlparser.parser.ParseException: Encountered unexpected token: "," ","

    at line 1, column 64.

Was expecting one of:

    "&"
    "::"
    "<<"
    ">>"
    "COLLATE"
    "["
    "]"
    "^"
    "|"

	at net.sf.jsqlparser.parser.CCJSqlParser.generateParseException(CCJSqlParser.java:26538)
	at net.sf.jsqlparser.parser.CCJSqlParser.jj_consume_token(CCJSqlParser.java:26377)
	at net.sf.jsqlparser.parser.CCJSqlParser.PrimaryExpression(CCJSqlParser.java:11047)
	at net.sf.jsqlparser.parser.CCJSqlParser.BitwiseXor(CCJSqlParser.java:10671)
	at net.sf.jsqlparser.parser.CCJSqlParser.MultiplicativeExpression(CCJSqlParser.java:10624)
	at net.sf.jsqlparser.parser.CCJSqlParser.AdditiveExpression(CCJSqlParser.java:10587)
	at net.sf.jsqlparser.parser.CCJSqlParser.BitwiseAndOr(CCJSqlParser.java:10533)
	at net.sf.jsqlparser.parser.CCJSqlParser.ConcatExpression(CCJSqlParser.java:10509)
	at net.sf.jsqlparser.parser.CCJSqlParser.SimpleExpression(CCJSqlParser.java:10495)
	at net.sf.jsqlparser.parser.CCJSqlParser.Upsert(CCJSqlParser.java:2428)
	at net.sf.jsqlparser.parser.CCJSqlParser.SingleStatement(CCJSqlParser.java:147)
	at net.sf.jsqlparser.parser.CCJSqlParser.Statement(CCJSqlParser.java:81)
	at net.sf.jsqlparser.parser.CCJSqlParserUtil.parseStatement(CCJSqlParserUtil.java:163)
	... 4 more
```

**Comment by siaron** [comment]
When to release?

**Comment by siaron** [comment]
current not support.

**Comment by wumpz** [comment]
Using 4.1-SNAPSHOT is supported.

**Comment by wumpz** [comment]
At the moment I am deploying the most recent version of JSqlParser 4.1-SNAPSHOT.

**Comment by manticore-projects** [comment]
Solved via PR #1105.
